samples/snippets/create_job_with_embedded_captions.py

```diff
@@ -0,0 +1,199 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Transcoder sample for creating a job that embeds captions in the output video.
+
+Example usage:
+    python create_job_with_embedded_captions.py --project_id <project-id> --location <location> \
+      --input_video_uri <uri> --input_captions_uri <uri> --output_uri <uri>
+"""
+
+# [START transcoder_create_job_with_embedded_captions]
+
+import argparse
+
+from google.cloud.video import transcoder_v1
+from google.cloud.video.transcoder_v1.services.transcoder_service import (
+    TranscoderServiceClient,
+)
+
+
+def create_job_with_embedded_captions(
+    project_id,
+    location,
+    input_video_uri,
+    input_captions_uri,
+    output_uri,
+):
+    """Creates a job based on an ad-hoc job configuration that embeds captions in the output video.
+
+    Args:
+        project_id (str): The GCP project ID.
+        location (str): The location to start the job in.
+        input_video_uri (str): Uri of the input video in the Cloud Storage
+          bucket.
+        input_captions_uri (str): Uri of the input captions file in the Cloud
+          Storage bucket.
+        output_uri (str): Uri of the video output folder in the Cloud Storage
+          bucket."""
+
+    client = TranscoderServiceClient()
+
+    parent = f"projects/{project_id}/locations/{location}"
+    job = transcoder_v1.types.Job()
+    job.output_uri = output_uri
+    job.config = transcoder_v1.types.JobConfig(
+        inputs=[
+            transcoder_v1.types.Input(
+                key="input0",
+                uri=input_video_uri,
+            ),
+            transcoder_v1.types.Input(
+                key="caption-input0",
+                uri=input_captions_uri,
+            ),
+        ],
+        edit_list=[
+            transcoder_v1.types.EditAtom(
+                key="atom0",
+                inputs=["input0", "caption-input0"],
+            ),
+        ],
+        elementary_streams=[
+            transcoder_v1.types.ElementaryStream(
+                key="video-stream0",
+                video_stream=transcoder_v1.types.VideoStream(
+                    h264=transcoder_v1.types.VideoStream.H264CodecSettings(
+                        height_pixels=360,
+                        width_pixels=640,
+                        bitrate_bps=550000,
+                        frame_rate=60,
+                    ),
+                ),
+            ),
+            transcoder_v1.types.ElementaryStream(
+                key="audio-stream0",
+                audio_stream=transcoder_v1.types.AudioStream(
+                    codec="aac", bitrate_bps=64000
+                ),
+            ),
+            transcoder_v1.types.ElementaryStream(
+                key="cea-stream0",
+                # The following doesn't work because "mapping" is a reserved
```

Comment:
Thanks for explaining this in the samples!

Reply:
No problem